src/Hamiltonians/TwoBodyCorrelation.jl

```diff
@@ -0,0 +1,70 @@
+# Methods that need to be implemented:
+#   •  starting_address(::AbstractHamiltonian) - not needed
+"""
+    TwoBodyCorrelation(d::Int)
+
+Implements the two-body correlation operator for `BoseFS2C` for computing the correlation function
+between the impurity and a boson from the Bose gas as a function of the distance `d`, which can take
+`Integer` values from `0` to `M` (the number of modes).
+It works on Fock states in momentum space and could return a `Complex` value.
```

Comment:
```suggestion
Implements the two-body correlation operator computing the correlation function
between the impurity and a boson from the Bose gas as a function of the distance `d`, which can take
`Integer` values from `0` to `M` (the number of modes).
It currently only works on [`BoseFS2C`](@ref) on Fock states in momentum space. It returns a `Complex` value.
```

Reply:
I would word it this way. In principle, we could implement the same thing for other Fock addresses.

Reply:
The funny part is, "It returns a `Complex` value with zero imaginary component", how can we word it nicely?